src/errors.ts

```diff
@@ -52,3 +52,13 @@ export class ClientStoppedError extends Error {
         super("MatrixClient has been stopped");
     }
 }
+
+export class UnsupportedEndpointError extends Error {
```

Comment:
Flyby comment: please give this class (and any other exported symbols) some [documentation]. What does it represent? How should downstream applications use it? What do the properties within it mean?

See https://github.com/element-hq/element-web/blob/develop/code_style.md#comments